source/includes/access-mongorestore-collections.rst

```diff
@@ -0,0 +1,7 @@
+To restore collection data to a database with authentication enabled,
+you must possess the appropriate user roles.
```

Comment:
Here and elsewhere rather than 'you must possess' probably 'the connecting user must possess' as in the mongoexport snippet